### PR TITLE
Rename CredentialMakeResult to MakeNewCredentialResult and adjust its namespace 

### LINKED ISSUE #432: Namespace inconsistency between CredentialMakeResult and AssertionVerificationResult
`CredentialMakeResult` should not be a class child of `Fido2NetLib.Fido2`. `AssertionVerificationResult` is at the root of the namespace `Fido2NetLib.Objects`. We should probably move `CredentialMakeResult` under the namespace `Fido2NetLib.Objects`.

### PR DESCRIPTION
**Pull request by joegoldman2**
Fixes #432

### CODE REVIEW

**Comment by iamcarbon** [comment]
@joegoldman2 Let's rename this type to MakeNewCredentialResult while we are here.

**Comment by iamcarbon** [comment]
👍

File: Src/Fido2.Models/Objects/MakeNewCredentialResult.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (aseigler) [inline Src/Fido2.Models/Objects/MakeNewCredentialResult.cs:5]: Some of these usings seem unused?

**Review by aseigler — CHANGES_REQUESTED** [review]

File: Src/Fido2.Models/Objects/MakeNewCredentialResult.cs (hunk 1 above)
Comment (joegoldman2) [reply Src/Fido2.Models/Objects/MakeNewCredentialResult.cs]: Good catch! In fact, they are all unused. I remove them.

**Review by abergs — APPROVED** [review]
LGTM. @aseigler anything or are we good to merge?

**Comment by aseigler** [comment]
> LGTM. @aseigler anything or are we good to merge?

I'm happy if you're happy!

**Pull request merged**